server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/python/PythonLanguagePlugin.kt

```diff
@@ -0,0 +1,85 @@
+package org.jetbrains.bsp.bazel.server.sync.languages.python
+
+import ch.epfl.scala.bsp4j.BuildTarget
+import ch.epfl.scala.bsp4j.BuildTargetDataKind
+import ch.epfl.scala.bsp4j.PythonBuildTarget
+import ch.epfl.scala.bsp4j.PythonOptionsItem
+import org.jetbrains.bsp.bazel.info.BspTargetInfo.FileLocation
+import org.jetbrains.bsp.bazel.info.BspTargetInfo.PythonTargetInfo
+import org.jetbrains.bsp.bazel.info.BspTargetInfo.TargetInfo
+import org.jetbrains.bsp.bazel.server.sync.BazelPathsResolver
+import org.jetbrains.bsp.bazel.server.sync.BspMappings
+import org.jetbrains.bsp.bazel.server.sync.dependencytree.DependencyTree
+import org.jetbrains.bsp.bazel.server.sync.languages.LanguagePlugin
+import org.jetbrains.bsp.bazel.server.sync.model.Module
+import java.net.URI
+import java.nio.file.Path
+import kotlin.io.path.toPath
+
+class PythonLanguagePlugin(
+    private val bazelPathsResolver: BazelPathsResolver
+) : LanguagePlugin<PythonModule>() {
+    private var defaultInterpreter: URI? = null
+    private var defaultVersion: String? = null
+
+    override fun prepareSync(targets: Sequence<TargetInfo>) {
+        val targetWithSdk = targets.filter { it.hasPythonTargetInfo() && it.pythonTargetInfo.hasInterpreter() }.firstOrNull()
+        val defaultTargetInfo = targetWithSdk?.pythonTargetInfo
```

Comment:
i'd just create a method "calculateDetultTargetInfo` and put there this line and the previous one